static/src/stylesheets/module/experiments/_survey.scss

```diff
@@ -0,0 +1,74 @@
+.impressions-survey__heading {
+  padding: .5em 1em;
+  font-size: .8em;
+  background-color: #aad7e4;
+}
+
+.impressions-survey__title {
+  display: inline-block;
+  font-family: 'Guardian Text Sans Web', 'Helvetica Neue', 'Helvetica', 'Arial', 'Lucida Grande', 'sans-serif';
```

Comment:
Should probably use one of our font mixins https://github.com/guardian/frontend/blob/master/static/src/stylesheets/pasteup/typography/_src.scss